### PR TITLE
Fix issue4

### PR DESCRIPTION
**Pull request by gittiver**
moved README to README.md (markdown format)
added section formatting and link to packaging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,4 +31,37 @@ to help out.
...
+
+- Use the configure script to configure doxymacs:
+
+   $ ./configure
+   $ make
+   $ make install
+
+  Use ./configure --help for help on customising your configuration.
+
+for details see [[INSTALL][INSTALL]]
```
Comment (pniedzielski) [inline README.md:67]: This looks like an org-mode link.  Could you change it to Markdown?  I'm wondering whether `[INSTALL](INSTALL)` will render correctly on the GitHub web interface, and this is a quick test with the Markdown change.

**Comment by pniedzielski** [comment]
This PR subsumes #12 , right?  If so, I'll go ahead and close that other PR.

Apart from the one thing mentioned inline in the review, LGTM

**Review by pniedzielski — COMMENTED** [review]
Apart from the one thing mentioned inline in the review, LGTM

File: README.md (hunk 1 above)
Comment (gittiver) [reply README.md]: Link changed to Markdown, works as expected. ;)
I experimented with org mode as this is mostly also rendered by GitHub.

**Comment by gittiver** [comment]
> This PR subsumes #12 , right? If so, I'll go ahead and close that other PR.

yes, #12 is included. just want to make sure that the README change could be done separately.

**Comment by pniedzielski** [comment]
Merged with 69b5cc2.  Thanks!

**Pull request closed without merging**